internal/model/bookmark.go

```diff
@@ -14,7 +14,8 @@ type BookmarkDTO struct {
 	Excerpt       string `db:"excerpt"       json:"excerpt"`
 	Author        string `db:"author"        json:"author"`
 	Public        int    `db:"public"        json:"public"`
-	Modified      string `db:"modified"      json:"modified"`
+	Created       string `db:"created_at"    json:"created"`
```

Comment:
Can we name this fields `CreatedAt` / `ModifiedAt` as well?

Reply:
For Golang code, `CamelCase`, for JSON `snake_case`. Though honestly we could just match everything to the same `CamelCase` in golang and avoid writting so many json annotations. What do you think?

Reply:
> @DesarrolloAntonio what do you think?

Regarding backward compatibility in the Shiori app, I have seen that there is an option to set one or more alternative names. I have never tried it, but theoretically, it should work.`    

```
@SerializedName("createdAt", alternate = ["created_at"])
val createdAt: String?,
```